README.md

```diff
@@ -1,19 +1,31 @@
-# Dangerous Dave Replica
+## Introduction
+If you don't already know why you need a monitoring and observability solution, it can be challenging to find an example that is illustrative, compelling, and engaging. There are literally thousands of "monitor this fake take-out food website" examples and - while they certainly show a valid real-world example of something you might want to monitor - they are usually a real yawner.
 
-This project is a replica of the 1988 DOS game Dangerous Dave, made by John Romero. The project was built in Python along with a team of three students (Arthur Medeiros, Guilherme Cattani and me), as a course assignment.
+What if - and hear us out - you could play around with a monitoring solution AND PLAY A VIDEO GAME at the same time?!? WE KNOW, RIGHT?!?
 
-The goal of the project was to study and practice the three types of programming paradigms: imperative, object-oriented and functional. To achieve this, we picked Python as a language since it can perform all three types of tasks in a fairly good way.
+That's what this example is all about. Install a game, set it up in New Relic, and then play the game to see your stats. 
+
+Dangerous Dave was a classic 1980's side-scroller style game that many of us spent far too many hours playing when we should have been working, doing homework, or being otherwise productive. Now we've turned the tables, making Dave help us with our actual work.
```

Comment:
I would argue that you and I are outliers, Leon. I would suggest changing "many of us" to simply "many" and "making Dave help us with our actual work" to "making Dave help with actual work."